config.js

```diff
@@ -37,5 +39,5 @@ try {
 } catch (e) {
 	//it is okay if no override file is provided
 }
-configInput.colors = getChalkColors(configInput, providedConfig);
+configInput.colors.logs = getChalkColors(configInput.colors, providedConfig.colors);

```

Comment:
I would rather have getChalkColors return an object that contains logs and lines..I know this is just a temporary hack, but I dislike it xD